### DISCUSSION TITLE
Splash screen support

### DISCUSSION THREAD

**Discussion by vzahradnik** [davetcc/tcMenu#386] [Q&A]
Hi Dave,

I'd like to create a splash screen which is shown for about 2-5 seconds once the firmware boots up.

First of all, what approach would you suggest? Taking over the display from TcMenu?

Second of all... Would it make sense to make this part of this library?

Thanks for the advice/opinion about this.

Vlad

**Comment by davetcc** [comment]
To be honest that’s more a function of the display library. You need to present the splash in the library initialisation routine for it to be early enough. I think it would be too late by the time tcMenu fired up. 

You could also use take over display, it’s commonly used for that kind of thing.

**Reply by vzahradnik** [reply]
I tried the display takeover method and it's not too late at all. It works great! I still have some issues with rendering images but this has nothing to do with TcMenu. I'm sure it will be a quick fix.

Thanks for the advice.
